components/HeaderNav/HeaderNav.js

```diff
@@ -10,7 +10,7 @@ export default function HeaderNav({ navigation }) {
       <View style={styles.navButtonContainer}>
         <View style={styles.navButton}><Button title='CHANGE DATE'/></View>
         <View style={styles.navButton}><Button onPress={() => navigation.navigate('CreateEventForm')}  title='CREATE EVENT'/></View>
-        <View style={styles.navButton}><Button title='LOG OUT'/></View>
+        <View style={styles.navButton}><Button onPress={() => navigation.navigate('Login')} title='LOG OUT'/></View>
```

Comment:
Cool! Now we have functionality to the log out button. Next, we need to implement the date picker to the "change date" button in the nav.

Reply:
For sure! And I think this is all the logout button is going to do for now since our MVP does not include user's logging in. If we get to that stretch goal then we can add more functionality to this button.